pkg/core/mempool/mem_pool.go

```diff
@@ -434,21 +444,22 @@ func (mp *Pool) checkPolicy(tx *transaction.Transaction, policyChanged bool) boo
 }
 
 // New returns a new Pool struct.
-func New(capacity int, payerIndex int, enableSubscriptions bool, updateMetricsCb func(int)) *Pool {
+func New(capacity int, payerIndex int, enableSubscriptions bool, enableSingleUseSubscriptions bool, updateMetricsCb func(int)) *Pool {
```

Comment:
This flag doesn't make much sense to me (performance difference is about zero). But we can optimize things later.